build/package/Microsoft.DotNet.Cli.Nupkg.targets

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Layout" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  
+  <Target Name="SetupGenerateNugetPackagesInputsOutputs"
+          DependsOnTargets="Init">
+    <PropertyGroup>
+      <NupkgVersionSuffix>$(CommitCount)</NupkgVersionSuffix>
+      <ProjectsSrcDirectory>$(RepoRoot)/src</ProjectsSrcDirectory>
+      <PackagingBuildBasePath>$(Stage2CompilationDirectory)/forPackaging</PackagingBuildBasePath>
+      <NupkgOutputDirectory>$(PackagesDirectory)</NupkgOutputDirectory>
+    </PropertyGroup>
+    
+    <ItemGroup>
+      <ProjectsToPack Include="$(ProjectsSrcDirectory)/Microsoft.DotNet.Cli.Utils" >
+        <ProjectName>Microsoft.DotNet.Cli.Utils</ProjectName>
+        <Version>$(VersionMajor).$(VersionMinor).$(VersionPatch)-preview3-$(CommitCount)</Version>
+      </ProjectsToPack>
+      <ProjectsToPack Include="$(ProjectsSrcDirectory)/Microsoft.DotNet.Compiler.Common">
+        <ProjectName>Microsoft.DotNet.Compiler.Common</ProjectName>
+        <Version>$(VersionMajor).$(VersionMinor).$(VersionPatch)-preview3-$(CommitCount)</Version>
+      </ProjectsToPack>
+      <ProjectsToPack Include="$(ProjectsSrcDirectory)/Microsoft.DotNet.Files">
+        <ProjectName>Microsoft.DotNet.Files</ProjectName>
+        <Version>$(VersionMajor).$(VersionMinor).$(VersionPatch)-preview3-$(CommitCount)</Version>
+      </ProjectsToPack>
+      <ProjectsToPack Include="$(ProjectsSrcDirectory)/Microsoft.DotNet.InternalAbstractions">
+        <ProjectName>Microsoft.DotNet.InternalAbstractions</ProjectName>
+        <Version>1.0.1-beta-$(CommitCount)</Version>
+      </ProjectsToPack>
+      <ProjectsToPack Include="$(ProjectsSrcDirectory)/Microsoft.DotNet.ProjectModel">
+        <ProjectName>Microsoft.DotNet.ProjectModel</ProjectName>
+        <Version>$(VersionMajor).$(VersionMinor).$(VersionPatch)-rc4-$(CommitCount)</Version>
```

Comment:
Can we provide logical names for these release variables? `preview3` seems to represent `ToolsVersionReleaseSuffix` while `rc4` represents `RuntimeVersionReleaseSuffix`? This would better capture the intent of the differences.